demo/portal/src/app/main.tpl.html

```diff
@@ -11,11 +11,11 @@ <h1 ng-click="navigateTo('app.home')">FeedHenry Work Force Management</h1>
 
     <md-sidenav ng-if="profileData" layout="column" class="md-sidenav-left md-whiteframe-z2" md-component-id="left" md-is-locked-open="$mdMedia('gt-sm')">
       <md-list>
-        <md-list-item ng-click="navigateTo('app.workorder')" ng-class="{active: $state.includes('app.workorder')}">
+        <md-list-item ng-if="authorised['workorder']" ng-click="navigateTo('app.workorder', {authorised: hasResourceRole('admin', 'workorder')})" ng-class="{active: $state.includes('app.workorder')}">
```

Comment:
[Antipattern] Template should not execute complex logic. 
`authorised: hasResourceRole('admin', 'workorder')}` should be mounted in controller.